.github/workflows/tests.yml

```diff
@@ -117,5 +118,26 @@ jobs:
       - name: Install Hatch
         run: pip install hatch==${{ env.HATCH_VERSION }}
 
+      - name: Install Ollama and pull the required models
```

Comment:
Does it make sense to have this step run only on integration tests?

Reply:
This step takes some time (about 1 minute), so I think it makes sense to skip it for unit tests that do not require an Ollama instance running.